moped-database/migrations/1648075868116_insert_moped_types/down.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM public.moped_types WHERE type_order BETWEEN '6' AND '9';
```

Comment:
🙏 for writing the migrations in an ID agnostic way and letting the DB sequencing stay in sync with the data itself. For this down migration, would you consider using the `type_name` field as the unique identifier for the records, since it's the name that is representing the real-world meaning of each record, not the order in which we choose to display them? Like, we wouldn't ever have two moped_types of "Fiber - New", so it's a good "unique" field to use. Doing this also helps with query readability, FWIW.

```
DELETE FROM public.moped_types
WHERE type_name in ('Fiber - Mod', 'Fiber - New', 'Fiber - Repair');
```

If not, would you consider explicitly defining the field values you want to use in an `IN` clause.  It's been my experience that `BETWEEN` is generally intended to be used to compare integers and to include (`<=` & `>=`) both values given to it as arguments. This query is using it here in string context, which interprets characters by their ASCII or Unicode values, maybe? I'm not sure, but `select 'R' BETWEEN '=' and 't';` evaluates to true. 🤷. The query also could match a record with `type_order` of `9`, which is not in the set of the records you're inserting into in the `up.sql` migration.